src/services/purchaseService.ts

```
@@ -333,29 +320,24 @@ const handleBoosterPackPurchase = async (
 ): Promise<IPurchaseResponse> => {
     const typeCategory = getStoreItemTypesCategory(typesName);
     logger.debug(`type category ${typeCategory}`);
     switch (typeCategory) {
-        default: {
-            const inventory = await getInventory(accountId);
-            const resp = await addItem(inventory, typesName, quantity);
-            await inventory.save();
-            return resp;
-        }
+        default:
+            return await addItem(inventory, typesName, quantity);
```

Biome (1.9.4)

[error] The default clause should be the last switch clause.

The following case clause is here:

Regardless its position, the default clause is always executed when there is no match. To avoid confusion, the default clause should be the last switch clause.

(lint/suspicious/useDefaultSwitchClauseLast)